hack/tools/go.mod

```diff
@@ -4,5 +4,6 @@ go 1.12
 
 require (
 	github.com/golangci/golangci-lint v1.18.0
-	sigs.k8s.io/controller-tools v0.2.0
+	sigs.k8s.io/cluster-api/hack/tools v0.0.0-20190926205936-6fc621a1c6af
+	sigs.k8s.io/controller-tools v0.2.1
```

Comment:
is this change necessary?

Reply:
It comes with CAPI, yes

Reply:
we're only importing tools for the release notes which does not depend on controller-tools

This change is unnecessary for this specific PR, but if you want to bump this version now you will also have to bump it in go.mod or change the CRD verification test

Reply:
`sigs.k8s.io/cluster-api/hack/tools` imports `sigs.k8s.io/controller-tools` v0.2.1, which is why it gets updated

Reply:
yeah, i saw that, but that doesn't mean we have to bump this version, the tools in this repo are still building on v0.2.0

Reply:
The controller-tools are only used in `hack/tools` as far as I can tell. Are you referring to the controller-runtime version?

Reply:
sorry, let me try again:

the crds are built (or, at least verified) using controller-tools v0.2.0
This update is unnecessary for the scope of this PR. Just because capi/tools uses controller-tools v0.2.0 does not mean we've made the update in this repo.

Can you update the title/description to include the CRD updates if the controller-tools dependency is also getting changed?

Reply:
ok reworded, and split the commits, ptal